### PR TITLE
Meta info update

### PR DESCRIPTION
**Pull request by MaksimChepelev**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ const initApp = function () {
   app.post('/api/v2/status', fogController.getFogControllerStatusEndPoint);
   app.get('/api/v2/instance/create/type/:type', fogController.fogInstanceCreateEndPoint);
 
-  // app.get('/api/v2/instance/getfabriclist', fogController.getFogListEndPoint);
```
Comment (ekrylovich) [inline src/server.js]: Did you change clients that consume this endpoints????

File: src/server/controllers/api/elementInstanceController.js [hunk 2]
```diff
@@ -195,11 +195,11 @@ import ElementInstanceToCleanUpService from "../../services/elementInstanceToCle
         name: 'bodyParams.name',
         logSize: 'bodyParams.logSize',
         config: 'bodyParams.config',
-        fogInstanceId: 'bodyParams.fabricInstanceId',
```
Comment (ekrylovich) [inline src/server/controllers/api/elementInstanceController.js]: Do you update the client>?

File: src/server.js (hunk 1 above)
Comment (MaksimChepelev) [reply src/server.js]: yes

File: src/server/controllers/api/elementInstanceController.js (hunk 2 above)
Comment (MaksimChepelev) [reply src/server/controllers/api/elementInstanceController.js]: yes

File: package.json [hunk 3]
```diff
@@ -12,10 +12,27 @@
...
+    "Darya Busel",
+    "Alexander Shpak",
+    "Kate Lukashick",
+    "Eugene Pankov",
+    "Maksim Chepelev",
+    "Ivan Panasiuk",
+    "Tetiana Yatsiuk"
+  ],
+  "license": "Copyright (c) 2018 Edgeworx, Inc. \nEclipse ioFog is provided under the Eclipse Public License 2.0 (EPL-2.0) \nhttps://www.eclipse.org/legal/epl-v20.html",
   "bugs": {},
```
Comment (ilaryionava) [inline package.json]: is this the place for this e-mail: edgemaster@iofog.org (for bug reporting?)

File: src/server.js (hunk 1 above)
Comment (ilaryionava) [reply src/server.js]: for example pureegde ?

File: src/server/controllers/api/elementInstanceController.js (hunk 2 above)
Comment (ilaryionava) [reply src/server/controllers/api/elementInstanceController.js]: check pureedge also

File: README.md [hunk 4]
```diff
@@ -22,15 +22,15 @@
...
+&emsp;&emsp;&emsp;[http://localhost:54421/api/v2/instance/create/type/:type?t=:userAccessToken](http://localhost:54421/api/v2/instance/create/type/:type?t=:userAccessToken)
 
 &emsp;&emsp;&emsp;where &#39;:type&#39; is FogType which can be 1 for Standard Linux (x86) OR 2 for ARM Linux 
 
 &emsp;&emsp;&emsp;and &#39;:userAccessToken&#39; is obtained by creating user
 
 - Fog provisioning
 
-&emsp;&emsp;&emsp;[http://localhost:3000/api/v2/authoring/fabric/provisioningkey/instanceid/:instanceId](http://localhost:3000/api/v2/authoring/fabric/provisionkey/instanceid/:instanceId)
+&emsp;&emsp;&emsp;[http://localhost:54421/api/v2/authoring/fabric/provisioningkey/instanceid/:instanceId](http://localhost:54421/api/v2/authoring/fabric/provisionkey/instanceid/:instanceId)
```
Comment (ilaryionava) [inline README.md]: haven't you changed this to iofog in url ??

File: src/server/controllers/api/elementInstanceController.js (hunk 2 above)
Comment (MaksimChepelev) [reply src/server/controllers/api/elementInstanceController.js]: done

File: src/server.js (hunk 1 above)
Comment (MaksimChepelev) [reply src/server.js]: done

File: src/server/controllers/api/elementInstanceController.js (hunk 2 above)
Comment (MaksimChepelev) [reply src/server/controllers/api/elementInstanceController.js]: done

**Comment by ilaryionava** [comment]
@MaksimChepelev resolve conflict and I think it can be merged ?

**Pull request merged**